### PR TITLE
0.2.0 integration PR

### PR DESCRIPTION
**Pull request by talonchandler**
Please excuse this hodge-podge PR full of integration fixes found during integration testing and debugging. 

- misc typos
- remove fluorescence from README figure
- remove final pre-processing vestiges
- Ziwen's automation scripts 
- Warnings if user has misconfigured micromanager (addressing #217)

### CODE REVIEW

File: recOrder/plugin/widget/main_widget.py [hunk 1]
```diff
@@ -2238,6 +2246,40 @@ def run_calibration(self):
 
         self.worker.start()
 
+    def _check_MM_config_setup(self):
```
Comment (talonchandler) [inline recOrder/plugin/widget/main_widget.py:2249]: This function isn't particularly beautiful, but none of our mmc code is and it's just for warnings.

**Comment by talonchandler** [comment]
I just tested the fixes for #217 and they're working as I expect. I will link that issue and move to testing the automation scripts.

**Comment by talonchandler** [comment]
The automation script is working well. Great job, Ziwen. 

I'm going to write up the docs for the new installation, then let's merge this.

File: recOrder/scripts/repeat-calibration.py [hunk 2]
```diff
@@ -5,9 +5,9 @@
 import matplotlib.pyplot as plt
```
Comment (ziw-liu) [inline recOrder/scripts/repeat-calibration.py:5]: Unused import?

Comment (talonchandler) [reply recOrder/scripts/repeat-calibration.py]: Good catch!

**Comment by talonchandler** [comment]
@ziw-liu  and I just finished a final round of testing. We are merging this branch and releasing its `HEAD` as `0.2.0`.

**Pull request merged**